io_scene_godot/converters/animation.py

```diff
@@ -34,23 +61,47 @@ def __init__(self):
         self['length'] = 0
         self.track_count = 0
 
+    def __hash__(self):
+        # magical hash value, it is safe as
+        # lru_cache is used as 'typed'
+        return 0x25161239850235
```

Comment:
I am dubious about this, as it means that every single AnimationResource has the same hash. If you leave this at default, I think it uses the instances memory address.

I guess I don't quite see the purpose of using a custom hash here.

Reply:
it is because lru_cache need all the parameter hashable, that is it will use the __hash__ hook